src/shared/Core/Gpg.cs

```diff
@@ -83,6 +83,9 @@ public void EncryptFile(string path, string gpgId, string contents)
                     throw new Trace2Exception(_trace2, "Failed to start gpg.");
                 }
 
+                // probably it should be implemented in the process manager
+                gpg.Start();
+
```

Comment:
The `null` check above only made sense when we were calling `Process.Start(...)` which returns `null` when internally, the `Start()` call was `false`.

To maintain behaviour we should probably do this instead:

```csharp
if (!gpg.Start())
{
    throw new Trace2Exception(_trace2, "Failed to start gpg.");
}
```

We should also do the same thing to fix the bug that existing just a bit further up in the `DecryptFile()` method.